### ISSUE TITLE
There are doubts in lines 44 to 49 of Astar.m. 

### ISSUE THREAD

**Issue by zhzhzhzhzhD** [PiuPiuXxx/PathPlanning-Matlab#1]
There are doubts in lines 44 to 49 of Astar.m. I think the nodes placed in the closed list will not be accessed and modified again.

**Comment by HYHJessica** [comment]
I agree with zhzhzhzhzhD，more accurately,is line "closed(BinA(open(:,1:2),children(i,1:2)),:)=children(i,:);", it should be'closed(BinA(closed(:,1:2),children(i,1:2)),:)=children(i,:);'
it has nothing to do with open set.我认为应该是更新closed集中的数据，此处与open集无关
